model/events/parse.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/onflow/flow-go/fvm/evm/types"
 	"github.com/onflow/flow-go/model/flow"
```

Comment:
I think EVM location prefix should go into model/flow/event.go

Reply:
Updated in https://github.com/onflow/flow-go/pull/5262/commits/dfa6eedaebe634cb05f5f3e8ecc57066e44dc657